File: WebAdmin/Models/Communique.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace WebAdmin.Models
{
    public class Communique
    {
        public int CommuniqueId { get; set; }
        [JsonIgnore] public ushort Type { get; set; }
        public Local CommuniqueLocal { get; set; }
        public List<SubCategory> CommuniqueSubCategory = new List<SubCategory>();
        public string Description { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: WebAdmin/src/Models/Communique.js
class Communique {
    Id;
    Type;
    Name;
    LocalId;
    Description;
    StartDate;
    EndDate;

    constructor(id = -1, type = -1, name = '', localId = -1,
    description = '', startDate = '', endDate = ''){
       this.Id = id;
       this.Type = type;
       this.Name = name;
       this.LocalId = localId;
       this.Description = description;
       this.StartDate = startDate;
       this.endDate = endDate;

    }
}